docs/dsl_inspec.md

```diff
@@ -252,3 +252,104 @@ control 'ssh-1' do
   end
 end
 ```
+
+# Using Ruby in InSpec
+
+The InSpec DSL is a Ruby based language. This allows you to be flexible with
+Ruby code in controls:
```

Comment:
Do we normally add line breaks in our Markdown?

Reply:
Good point, probably needs some cleanup. It renders all the same but i have to review the style on a few things ty!